org.mwc.debrief.pepys/src/main/java/org/mwc/debrief/pepys/view/PepysImportView.java

```diff
@@ -348,6 +348,8 @@ public void widgetDisposed(final DisposeEvent e) {
 		treeGrid.verticalAlignment = GridData.FILL;
 		treeGrid.grabExcessVerticalSpace = true;
 		treeGrid.horizontalSpan = 5;
+		treeGrid.widthHint = 850;
```

Comment:
@saulhidalgoaular - does this fix the `blank panel` issue? 

That's a sneaky bug. Well spotted :-)

Reply:
Good afternoon @IanMayo . Actually, It was the setMinimumSize. I increased size of the TreeGrid section to increase the size of the panel, because the setSize of SWT was being ignored....